src/index.ts

```diff
@@ -44,13 +43,6 @@ async function writeFile(dtsCreator: DtsCreator, cssFile: string): Promise<void>
   // clears cache so that watch mode generates update-to-date typing.
   const content = await dtsCreator.create(cssFile, undefined, true);
   await content.writeFile();
-  if (content.messageList.length > 0) {
```

Comment:
@gdelmas why remove this?

Reply:
the new version of  `typed-css-modules` no longer has a `messagesList`.

i guess the reason is, that it was only used to collect classnames that could not be emitted. yet the new version can emit all classnames now.

i therefore also updated the definition to reflect that in line 26.

Reply:
@gdelmas Sounds good. I will merge the PR if you can rebase.